Examples/Swift/LocationPrivacyExample.swift

```diff
@@ -0,0 +1,46 @@
+import Mapbox
+
+@objc(LocationPrivacyExample_Swift)
+
+class LocationPrivacyExample_Swift: UIViewController, MGLMapViewDelegate {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        mapView.userTrackingMode = .follow
+
+        view.addSubview(mapView)
+    }
+
+    /**
+        In order to enable the alert that requests temporary precise location, please add the following key to your info.plist
+        Privacy - Location Temporary Usage Description Dictionary
+
+        You must then add MGLAccuracyAuthorizationDescription as key in the Privacy - Location Temporary Usage Description Dictionary
+     */
```

Comment:
The lines in this comment section have a lot of characters - when this eventually gets injected into the website ([example shown here](https://docs.mapbox.com/ios/maps/examples/add-marker-symbol/)), it could require the viewer to horizontally scroll a lot to be able to see them. I recommend breaking this into additional lines.